data-products/example_prison_data_product/application/transform.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+import logging
+import boto3
+import yaml
+import os
+from pathlib import Path
+
+
+logging.basicConfig()
+s3_client = boto3.client("s3")
+
+
+def get_data(bucket: str, key: str) -> pd.DataFrame:
+    response = s3_client.get_object(Bucket=bucket, Key=key)
+
+    status = response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+
+    if status == 200:
+        logging.info("Data has been collected from {}/{}".format(bucket, key))
+        return pd.read_csv(response.get("Body"))
+    else:
+        logging.error(
+            "Bucket {} or file {} does not exist".format(bucket, key))
+        raise
+
+
+def get_tables(bucket, key, source_data):
+    yaml_key = os.path.join(
+        "code/", key.split("/")[1], "extracted/metadata/02-data-dictionary.yml"
```

Comment:
```suggestion
        "code", key.split("/")[1], "extracted", "metadata", "02-data-dictionary.yml"
```
can you comment what key.split("/")[1] is or make a new variable for it which names it

Reply:
yep agree. Meant to change this split to use path. Have addressed